docs/application/native/guides/ui/dali/capture.md

```diff
@@ -0,0 +1,66 @@
+# Capture
+
+
+The Dali::Capture is a class to capture a snapshot of the current scene. The captured result can be saved as a file and rendered by DALi `ImageView` or `ImageVisual`.
+
+
+## How to capture your scene
+
+Applications should follow the example below to use `Dali::Capture`.
+
+``` C++
+Capture capture = Capture::New();
+
+capture.Start( source, size, path, clearColor, quality);
+```
+
+Each parameter of `Capture::Start` is denoted as follows:
+
+ * **Actor source**: source actor to be captured. Only this source actor and its children are captured.
+ * **Vector2 size**: size of captured result image.
+ * **std::string path**: system path to save the captured image file. If the path is empty, `Dali::Capture` does not save the captured image as a file. Instead, the captured image can be rendered by DALi `ImageView` or `ImageVisual`.
+   * `Dali::Capture` supports PNG and JPEG format for the saved image.
+ * **Vector4 clearcolor**: background color of the captured scene
+ * **int quality**: a value to control the quality of the saved image file. For JPEG file format, the value must be in the range [1,100]. Default value is 100.
+
+If required, You can also connect a callback function to a finished signal:
+
+``` C++
+capture.FinishedSignal().Connect(this, &CaptureSceneExample::OnCaptureFinished);
+
+void CaptureSceneExample::OnCaptureFinished( Capture capture, Capture::FinishState state )
+{
+  if ( state == Capture::FinishState::SUCCEEDED )
+  {
+    // Do something
+  }
+  else
+  {
+    // Do something
+  }
+}
+```
+
+You can also use the captured result as an image source to be rendered by DALi `ImageView` or `ImageVisual`.
+
+``` C++
+void CaptureSceneExample::OnCaptureFinished( Capture capture, Capture::FinishState state )
+{
+  if ( state == Capture::FinishState::SUCCEEDED )
+  {
+    std::string url = Toolkit::Image::GenerateUrl( capture.GetNativeImageSource() );
+    Toolkit::ImageView imageView = Toolkit::ImageView::New( url );
+
+    mRootLayer.Add( imageView );
+  }
+  else
+  {
+    // Do something
+  }
+}
+```
+
+## Related Information
+- Dependencies
+  - Tizen 4.0 and Higher for Wearable
+  - Tizen 6.0 and Higher for Mobile
```

Comment:
https://github.com/Samsung/tizen-docs/pull/1184#discussion_r459821557

>Thank you I changed the target branch to tizen_6.0_prepare.
And, yes, these Capture APIs have existed since Tizen 4.0 for Wearable.(Though we added several new APIs in this time)
The two ACR, first one is to copy existing APIs in wearable to mobile profile and, the last one is to add totally new APIs for both profiles.

Related with this, I'd like to share you my opinion about the publish date of this page..

tizen_6.0_prepare will be merged into master when Tizen 6.0 is published.
It means, if we use tizen_6.0_prepare, this update will not be published until Tizen 6.0 open date.
We don't have any issue with that in case of Mobile, as Mobile supports capture since 6.0.

In case of Wearable,
you said that the Capture API has been exist since Tizen 4.0 for Wearable.
However, as this page is both for mobile and wearable, 
even Tizen 4.0 Wearable supports this API, I'll postpone the publish of this page util Tizen 6.0 publish.

Is it OK that we hold the publish of this page until Tizen 6.0 publish date?

Reply:
If publish means merge done but not open until Tizen 6.0 is released, it's OK no problem!